src/authentication/team-token.ts

```diff
@@ -0,0 +1,58 @@
+import * as Knex from 'knex';
+import * as moment from 'moment';
+
+const randomstring = require('randomstring');
+
+export const teamTokenLength = 7;
+
+export interface TeamToken {
+  teamId: number;
+  token: string;
+  createdAt: number | moment.Moment;
+}
+export function teamTokenQuery(db: Knex, token?: string, teamId?: number) {
```

Comment:
Added documentation in the code. The optional arguments are actually optional, they just filter the set of all team-tokens.